### DISCUSSION TITLE
eza

### DISCUSSION THREAD

**Discussion by txtsd** [catppuccin/catppuccin#2528] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Now that [eza](https://github.com/eza-community/eza) (A modern alternative to ls) has [custom themes support](https://github.com/eza-community/eza/releases/tag/v0.19.2) via a theme.yml, it would be nice to have official catppuccin themes for it!

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Is there a benefit to the theme file over setting, for example, `LS_COLORS=$(vivid generate catppuccin-mocha)` where vivid is https://github.com/sharkdp/vivid

**Reply by shyun3** [reply]
I believe `LS_COLORS` only affects file and folder names. `eza` allows permissions, users, date modified, etc. to be customized as well.

For example, here is the default output on my machine:
<img width="366" height="212" alt="image" src="https://github.com/user-attachments/assets/870c9557-1485-4405-8284-dd1e203c7c26" />

Using `LS_COLORS=$(vivid generate catppuccin-mocha)`:
<img width="367" height="213" alt="image" src="https://github.com/user-attachments/assets/c421d3ea-46ac-4783-9127-053a766eb9da" />

Using the community catppuccin theme (see https://github.com/catppuccin/catppuccin/discussions/2528#discussioncomment-14025025):
<img width="369" height="215" alt="image" src="https://github.com/user-attachments/assets/39ef01e1-734f-4ab6-a29d-356fa9bda91d" />

**Comment by shyun3** [comment]
There is already a theme in the [eza-themes](https://github.com/eza-community/eza-themes) repo. However, there is only one flavor and it's not clear which one it is.

It would be nice if all flavors were available.

**Comment by ankddev** [comment]
I'm working on this!

**Reply by sgoudham** [reply]
It would be great if you could use [catppuccin/whiskers](https://github.com/catppuccin/whiskers) to generate the theme, will make the review process much faster!

**Comment by ankddev** [comment]
I created theme, but can somebody please make screenshots of it and open PR? Right now I haven't access to my PC and don't think that screenshots from Termux will be pretty.
https://github.com/ankddev/eza

**Comment by shyun3** [comment]
It looks like this has been officially released at https://github.com/catppuccin/eza so this discussion could be closed.
